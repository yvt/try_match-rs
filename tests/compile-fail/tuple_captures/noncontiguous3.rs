fn main() {
    let _ = try_match::try_match!(Some((1, 2)), Some((_1, _2)));
}
