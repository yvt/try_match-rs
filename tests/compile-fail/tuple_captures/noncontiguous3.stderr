error: non-contiguous tuple binding: `_0` is missing
 --> tests/compile-fail/tuple_captures/noncontiguous3.rs:2:55
  |
2 |     let _ = try_match::try_match!(Some((1, 2)), Some((_1, _2)));
  |                                                       ^^
